### DISCUSSION TITLE
Adonisjs Build Error

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#2936] [Help]
When I am trying to build my adonis app, I am receiving this error. 

**Uncaught exception: TypeError: asynck_hook_1.AsyncLocalStorage is not consturctor at Object.<ananymous>**

Can anyone help me?

**Comment by RomainLanz** [comment]
Hey @feniljariwala82! 👋 

What is your Node.js version?

**Reply by feniljariwala82** [reply]
14.17.2 !

**Comment by thetutlage** [comment]
I doubt that you have shared the actual error, coz there are many typos in the error you have shared. Please share the exact error with a screenshot

**Reply by feniljariwala82** [reply]
For your reference, I have converted my Adonis App into Electron App. Electron Distribution works fine, but when I try to open the app it shows this error.

![typo](https://user-images.githubusercontent.com/42708686/126490928-f685bc85-c7e3-4f2d-b5ef-c529aea659b7.png)
